src/portal_visualization/builders/anndata_builders.py

```diff
@@ -18,14 +17,14 @@ class RNASeqAnnDataZarrViewConfBuilder(ViewConfBuilder):
     https://portal.hubmapconsortium.org/browse/dataset/e65175561b4b17da5352e3837aa0e497
     """
 
-    def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
-        super().__init__(entity, groups_token, assets_endpoint, **kwargs)
+    def __init__(self, entity, groups_token, assets_endpoint):
+        super().__init__(entity, groups_token, assets_endpoint)
         # Spatially resolved RNA-seq assays require some special handling,
         # and others do not.
         self._is_spatial = False
         self._scatterplot_w = 9
 
-    def get_conf_cells(self, marker=None):
+    def get_configs(self, marker=None):
```

Comment:
The Builder subclasses now define `get_configs` instead of `get_conf_cells`, which is now defined only on the base class. The idea is to factor up behavior which is shared between subclasses.